### DISCUSSION TITLE
vps这两天突然延迟很高

### DISCUSSION THREAD

**Discussion by Sean19899999** [XTLS/Xray-core#4092] [Performance]
按照小小白白话文，使用自租的vps搭建的v2ray服务，连接方式好像是“xtls-rprx-vision”+“TLS”？（小白不懂，轻喷），挂靠了域名。之前一直使用非常流畅（已经用1年半了），但是这两天突然很卡，各种网站能打开但是就是转很久的圈圈，在v2rayN服务端测试ping是300左右，真连接延迟测试700左右（有时候还会失败变成-1）。请问可能是什么原因，如何捉虫啊？

**Comment by SpeedPartner** [comment]
去运营商机房捉,也可以去海底捉

**Comment by Sean19899999** [comment]
请问：机房说他们那边测试300左右，到咱们这700是正常的吗？另外海底是哪（小白不懂哈哈）在 2024年12月2日，10:47，SpeedPartner ***@***.***> 写道：﻿
去运营商机房捉,也可以去海底捉

**Comment by SpeedPartner** [comment]
使用路由追踪,自己测试到哪个机房节点延迟异常

**Comment by xiaorong61** [comment]
ping.pe 看看，可能是丢包
